### ISSUE TITLE
feature request - integrate open data from dwd

### ISSUE THREAD

**Issue by tanrak** [Gitsaibot/AF-Weather-Widget#22]
First, thank you for this simple and useful widget.
It would be great if the open data from the DWD (German Meteorological Service) could be integrated. Germany is covered by MET, but the data from DWD is a bit more accurate. If this is not completely out of the scope of this project, then maybe [this](https://brightsky.dev/) could be helpful.

**Comment by vinibali** [comment]
I agree with tanrak. Unfortunately in Hungary, the currently used Norwegian data is so inaccurate. There are used to give bad forecasts especially in rainy weather. Other local sites mostly works better. I haven't check the source code yet, is it something big?
Best regards

**Comment by vinibali** [comment]
Another day, when met.no showed around 15mm rain in the morning and we've got 5mm in the night. I'll let them know this, maybe there are some adjustments they could archive
